File: src/main/java/org/tud/oas/accessibility/PopulationAccessibility.java
package org.tud.oas.accessibility;

import java.util.ArrayList;
import java.util.List;

public class PopulationAccessibility {
    public List<Integer> ranges;

    PopulationAccessibility() {
        this.ranges = new ArrayList<Integer>();
    }

    void addRange(int range) {
        this.ranges.add(range);
    }
}

File: src/accessibility/PopulationAccessibility.cs
using System;
using System.Collections.Generic;

namespace DVAN.Accessibility
{
    public class PopulationAccessibility {
        public List<int> ranges;

        public PopulationAccessibility() {
            this.ranges = new List<int>();
        }

        public void addRange(int range) {
            this.ranges.Add(range);
        }
    }
}